lib/kube/proxy/websocket_client_testing.go

```diff
@@ -350,7 +349,11 @@ func (e *wsStreamClient) stream(conn *gwebsocket.Conn, options clientremotecomma
 	}()
 
 	wg.Wait()
-	close(errChan)
+	// always expect an error from the errChan since it means that the connection was closed
+	// by the server by sending a streamErr with the error.
+	// If no error happened during the remote execution, the server sends a streamErr with
+	// status = Success field.
+	// If the server didn't send any error, this call will block forever.
```

Comment:
Should we try to detect this and give a better error, so we don't rely solely on `go test -timeout` in case it fails this way?

Reply:
Introduced in b0b5212808dbad52f122844525ac897df8386a27